newsplease/config/config.cfg

```diff
@@ -263,6 +263,13 @@ end_date = '2999-12-31 00:00:00'
 strict_mode = False
 
 
+[ContentFilter]
+
+filter_predicates = {'title': 'cloud computing',
+                     'text': 'google | amazon'}
+pass_condition = title or text
```

Comment:
Please specify as a comment any other values that one can put here besides title or text.